src/website/webview/views.py

```diff
@@ -598,7 +609,18 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
             elif new_status == "pending" and undo_status_change:
                 suggestion.status = CVEDerivationClusterProposal.Status.PENDING
 
-            if old_status != new_status:
+            suggestion.save()
+
+        if status_change and new_status == "published":
+            with transaction.atomic():
+                tracker_issue = suggestion.create_nixpkgs_issue()
+                tracker_issue_link = request.build_absolute_uri(
+                    reverse("webview:issue_detail", args=[tracker_issue.code])
+                )
+                gh_issue_link = create_gh_issue(
```

Comment:
We'd probably need at least some rudimentary error handling here, otherwise it will be inscrutable what happend -- I suppose on GH failure simply nothing will change?

Reply:
Currently yeah, nothing changes thanks to the transaction. What do you suggest we do on error? For anything non-trivial in the view, I propose to post-pone it to a follow-up PR.